Assets/Scripts/TorchManager.cs

```diff
@@ -7,6 +7,8 @@ public class TorchManager : MonoBehaviour
     [SerializeField] private bool isTorchOn = false;
     [SerializeField] private Light TorchLight;
 
```

Comment:
You don't have to change it but... Menager is use mostly to menage many objects, so if it is just one torch you can name this script just Torch